package.json

```diff
@@ -55,6 +55,7 @@
     "mqtt": "^5.11.0",
     "prettier": "^3.0.0",
     "prettier-plugin-sh": "^0.17.4",
+    "react-native-webview": "^13.15.0",
```

Comment:
This is a heavy dependency. We only use it for types. Bundler will most likely treeshake this, but I'd rather not have this dependency tree in the project (audit, ...).